pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "opendrift-leeway"
+description = "Web GUI for Leeway simulations with OpenDrift"
+version = "2022.12.0"
```

Comment:
I think the version specifier was missing. Are you fine with calendar versioning like we use for our other Django projects?

Reply:
Sure, the format works for me.